### PR TITLE
Reverse broadcast-slashing flag to disable-broadcast-slashings

### PR DESCRIPTION
**Pull request by 0xKiwi**
**What type of PR is this?**
Flag reversal

**What does this PR do? Why is it needed?**
This PR deprecates the `--broadcast-slashing` flag, and introduces a `--disable-broadcast-slashings` that blocks slashings that are submitted into the beacon pool from being propagated. Reversing allows every node with a slasher to by default, propagate the slashings submitted.

### CODE REVIEW

**Comment by terencechain** [comment]
Test failed @0xKiwi

File: beacon-chain/rpc/beacon/slashings_test.go [hunk 1]
```diff
@@ -4,16 +4,19 @@ import (
 	"context"
 	"testing"
 
+	"github.com/prysmaticlabs/prysm/shared/featureconfig"
+
```
Comment (terencechain) [inline beacon-chain/rpc/beacon/slashings_test.go:8]:
```suggestion
```

File: shared/featureconfig/config.go [hunk 2]
```diff
@@ -197,9 +197,9 @@ func ConfigureBeaconChain(ctx *cli.Context) {
 		log.Warn("Enabling state reference copy")
 		cfg.EnableStateRefCopy = true
 	}
-	if ctx.Bool(broadcastSlashingFlag.Name) {
+	if ctx.Bool(disableBroadcastSlashingFlag.Name) {
 		log.Warn("Enabling broadcast slashing to p2p network")
```
Comment (terencechain) [inline shared/featureconfig/config.go:201]: This log message doesn't seem right

Comment (0xKiwi) [reply shared/featureconfig/config.go:201]: Good catch, thank you

**Comment by rauljordan** [comment]
PTAL @terencechain

**Pull request merged**